frontend/src/components/activity/dialog/FormScheduleEntryItem.vue

```diff
@@ -1,68 +1,98 @@
-          vee-rules="required|greaterThanOrEqual_date:@startDate"
-          :min="localScheduleEntry.start"
-          :allowed-dates="dateIsInSelectedPeriod"
-          :filled="false"
-          class="float-left date-picker mr-3 mt-1"
-          required
-        />
-
-        <e-time-picker
-          v-model="localScheduleEntry.end"
-          path="endDatetime"
-          :vee-rules="endTimeValidation"
-          :min="minEndTime"
-          :filled="false"
-          class="float-left mt-1 time-picker"
-          required
-        />
-      </v-col>
-
-      <v-col cols="1" class="pt-3 text-center">
-        <button-delete v-if="deletable" icon-only @click="$emit('delete')" />
-      </v-col>
-    </v-row>
-  </v-container>
+  <div class="ec-form-schedule-entry-item px-3 w-100">
+    <e-date-picker
+      v-model="localScheduleEntry.start"
+      value-format="YYYY-MM-DDTHH:mm:ssZ"
+      path="startDate"
+      :vee-id="'startDate' + startUTC"
+      vee-rules="required"
+      :allowed-dates="dateIsInAnyPeriod"
+      :filled="false"
+      class="area-startdate float-left date-picker mr-1 mt-md-1"
+      required
+    />
+
+    <div class="area-starttime">
+      <e-time-dropdown
+        v-model="localScheduleEntry.start"
+        path="startDatetime"
+        vee-rules="required"
+        :filled="false"
+        input-class="float-left mt-md-1"
+        required
+        value-format="YYYY-MM-DDTHH:mm:ssZ"
+        :items="startTimeList"
+        :menu-props="{
+          maxHeight: '320',
+          offsetOverflow: true,
+          offsetY: true,
+          nudgeLeft: '16px',
+        }"
+      />
+    </div>
+    <div class="area-dash mt-2">
+      <div class="text-field-alignment">-</div>
+    </div>
+    <div class="area-endtime">
+      <e-time-dropdown
+        v-model="localScheduleEntry.end"
+        value-format="YYYY-MM-DDTHH:mm:ssZ"
+        path="endDatetime"
+        :vee-rules="endTimeValidation"
+        :min="minEndTime"
+        :filled="false"
+        input-class="float-left mt-md-1 mr-1"
+        required
+        :items="endTimeList"
+        :menu-props="{
+          maxHeight: '320',
+          offsetOverflow: true,
+          offsetY: true,
+          nudgeLeft: '16px',
+        }"
+      >
+        <template #item="{ item }">
+          <span class="tabular-nums">{{ item.label }}</span
+          >&nbsp;<span class="opacity-60">({{ item.duration }})</span>
+        </template>
+      </e-time-dropdown>
+    </div>
+
+    <e-date-picker
+      v-model="localScheduleEntry.end"
+      value-format="YYYY-MM-DDTHH:mm:ssZ"
+      path="endDate"
+      :vee-rules="'required|greaterThanOrEqual_date:@startDate' + startUTC"
+      :min="localScheduleEntry.start"
+      :allowed-dates="dateIsInSelectedPeriod"
+      :filled="false"
+      :class="{ 'hide-control': $vuetify.breakpoint.mdAndUp && isSameDay }"
+      class="area-enddate float-left date-picker mr-3 mt-md-1"
+      required
+    />
+    <e-text-field
```

Comment:
![image](https://github.com/user-attachments/assets/bb11c100-07e4-4cc3-b761-29760b73b054)

this is a little off on mobile